sootup.core/src/main/java/sootup/core/jimple/basic/Local.java

```diff
@@ -153,6 +154,20 @@ public List<Stmt> getDefsForLocalUse(StmtGraph<?> graph, Stmt stmt) {
     return defStmts;
   }
 
+  public List<Stmt> getStmtsUsingthisLocal(Collection<Stmt> stmts, Stmt removedStmt) {
+    List<Stmt> localOccurrences = new ArrayList<>();
+    for (Stmt stmt : stmts) {
+      if (stmt.equivTo(removedStmt)) continue;
+      List<Value> stmtUses = stmt.getUsesAndDefs().collect(Collectors.toList());
```

Comment:
Do you really want `stmt.getUsesAndDefs()`? Shouldn't it be `stmt.getUses()`?

Reply:
getUses only gives me only the right hand side values like b = virtualinvoke(a) but stmt.getUsesAndDefs() gives me a = anything.

Reply:
Done